SEP-0008.md

```diff
@@ -0,0 +1,179 @@
+# SEP-0008 -- SunPy Core Library Versioning Scheme
+
+| SEP           | 0008                                 |
+|---------------|--------------------------------------|
+| title         | SunPy Core Library Versioning Scheme |
+| author(s)     | Stuart Mumford                       |
+| contact email | stuart@cadair.com                    |
+| date-creation | 2017-09-26                           |
+| type          | process                              |
+| status        | discussion                           |
+| discussion    |                                      |
+
+# Introduction
+
+This SEP describes the versioning scheme to use for the SunPy core library.
+Affiliated package maintainers can choose to adopt this scheme or develop their
+own.
+
+This scheme is derived from 
+[Semantic Versioning 2.0.0](http://semver.org/spec/v2.0.0.html) and 
+[PEP 440](https://www.python.org/dev/peps/pep-0440/), the main modifications
+from the Semantic Versioning specification is the handling of pre and
+development releases that follow the specification in PEP 440 to better
+integrate with Python packaging tools. This scheme also excludes some components
+of PEP 440 and Semantic Versioning, as they are not used for SunPy.
+
+
+# Version Numbering Specification
+
+## Summary
+
+The version number for SunPy will be of the following format:
+
+    MAJOR.MINOR.PATCH[{a|b|rc}N][.devM]
+
+The MAJOR, MINOR, and PATCH numbers shall be incremented in the following manner:
+
+1. MAJOR version when incompatible API changes are made.
```

Comment:
can you define API? I assume you mean public-facing API and not developer-facing API (for example how to register a new downloader to FIDO).

Reply:
API is defined later in the document.